cloud/pkg/controller/config/buffer.go

```diff
@@ -46,4 +55,26 @@ func init() {
 		QuerySecretBuffer = qsb
 	}
 	log.LOGGER.Infof("query secret buffer: %d", QuerySecretBuffer)
+
+	if peb, err := config.CONFIG.GetValue("pod-event-buffer").ToInt(); err != nil {
```

Comment:
perhaps we need to replace 'pod-event-buffer' into 'controller.pod-event-buffer'
The below config should be that too.

Reply:
ok